frontend/src/metabase/admin/settings/selectors.js

```diff
@@ -428,22 +425,8 @@ export const ADMIN_SETTINGS_SECTIONS = {
     key: "enable-embedding",
     name: t`Embedding`,
     order: 100,
-    settings: [
-      {
-        key: "enable-embedding",
-        display_name: t`Embedding`,
-        description: null,
-        widget: EmbeddingSwitchWidget,
-      },
-      {
-        key: "-static-embedding",
-        widget: StaticEmbeddingOptionCard,
-      },
-      {
-        key: "-interactive-embedding",
-        widget: InteractiveEmbeddingOptionCard,
-      },
-    ],
+    component: EmbeddingSettings,
```

Comment:
using `component` API [following Ryan's suggestion.](https://www.notion.so/Tech-Feature-flag-and-settings-for-SDK-9945e76d7e80404fa79b7d5b93049978?d=c507d778272445ab886b722db44cea1d&pvs=4)

Reply:
❤️